### DISCUSSION TITLE
Not found - What would be the reason ?

### DISCUSSION THREAD

**Discussion by lironsam** [filamentphp/filament#1027] [Help] [answered]
Hi,

I have successfully created several resources and I can without problems, edit or create.
However, for some reason I can't figure out, there is a resource I created called "Asset" that is linked to my asset table where I can easily display the list of all assets but I can't create nor edit them. If I want to create a new one or edit one, I get a 404 - not found error message.

After doing `php artisan route:optimize` and listing all the resources, the _"create"_ and _"edit"_ routes for the asset exist.

Is there a reason why there is "not found" for only create and edit?

To clarify, I have done the same manipulation for several tables and everything works. It is only with this one "Assets" that it does not work.

Thank you in advance,

**Comment by danharrin** [comment, marked as answer]
Hey! This is actually because there's already an '/assets` route that we use to serve CSS & JS in Filament.

The quick fix would be to add a custom `$slug` to your resource:

```php
protected ?string $slug = 'assets-resource';
```

But maybe we should think about renaming our `/assets` route to something else, since Asset is a common model.

**Reply by lironsam** [reply]
Yet it worked ! Just I needed to add the "static" keyword => ` protected static ?string $slug = 'assets-resource';`
Thank again !

**Comment by danharrin** [comment]
I know you've fixed this issue - but from the next version, you should be able to use `/assets` like normal - I moved the CSS and JS files to `/core/assets`.

**Reply by lironsam** [reply]
great ! Thank !
